.ci/pytorch/test.sh

```diff
@@ -1552,10 +1552,6 @@ test_executorch() {
 
   popd
 
-  # Test torchgen generated code for Executorch.
-  echo "Testing ExecuTorch op registration"
-  "$BUILD_BIN_DIR"/test_edge_op_registration
```

Comment:
This is already removed as part of https://github.com/pytorch/pytorch/pull/154165